verification/curator-service/api/src/geocoding/geocoder.ts

```diff
@@ -27,7 +27,13 @@ export enum Resolution {
     Country = 'Country',
 }
 
+export interface GeocodeOptions {
```

Comment:
Do we anticipate adding more, here? On the subject of adding this interface, vs. using a single param.

Reply:
seeing how many params the mapbox GeocodeRequest has I suspect we will add more yes, thus the options struct